src/components/global/Playground/index.tsx

```diff
@@ -38,21 +38,40 @@ const CodeBlockButton = ({ language, usageTarget, setUsageTarget, setCodeExpande
   );
 };
 
+type MdxContent = () => {};
+
+interface OutputTargetOptions {
```

Comment:
What is an "output target option" and why is it needed?

Reply:
I could probably call this `UsageTargetOptions` to align with our existing variable naming. We basically have two valid type signatures that can be assigned to the `code` prop. MDX content for single-file examples and an object for multi-file examples + advanced configurations. Having the types separate helps so I can later type-cast and have type checking for the advanced configuration options. This interface could use a comment on its purpose 👍